mixins/localize/localize-mixin.js

```diff
@@ -116,9 +90,13 @@ export const _LocalizeMixinBase = dedupeMixin(superclass => class LocalizeMixinC
 		const translatedMessage = new IntlMessageFormat(value, language);
 		let formattedMessage = value;
 		try {
-			if (Object.values(params).some(v => typeof v === 'function')) throw 'localize() does not support rich text.';
+			validateMarkup(formattedMessage, noAllowedTagsRegex);
```

Comment:
We do actual validation on `localize` messages now instead of just looking for replacements.